e2e/test/scenarios/dashboard-filters/reproductions/38245-negative-dashcard-id-switching-tabs.cy.spec.js

```diff
@@ -0,0 +1,82 @@
+import {
+  editDashboard,
+  getDashboardCard,
+  goToTab,
+  openQuestionsSidebar,
+  popover,
+  restore,
+  sidebar,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+const TAB_1 = {
+  id: 1,
+  name: "Tab 1",
+};
+
+const TAB_2 = {
+  id: 2,
+  name: "Tab 2",
+};
+
+const DASHBOARD_TEXT_FILTER = {
+  id: "3",
+  name: "Text filter",
+  slug: "filter-text",
+  type: "string/contains",
+};
+
+describe("issue 38245", () => {
```

Comment:
```suggestion
describe.skip("issue 38245", () => {
```

Reply:
I couldn't marge it without skip 😄 tests do not allow to do it